.github/workflows/e2e_getdents.yml

```diff
@@ -48,7 +48,7 @@ jobs:
           just get-credentials
       - name: Build and prepare deployments
         run: |
-          just node-installer
+          just microsoft-node-installer
```

Comment:
please change to node-installer-microsoft, which is more intuitive for tab completion.